packages/client-utils/test/snapshots/exports.test.js.md

```diff
@@ -10,6 +10,7 @@ Generated by [AVA](https://avajs.dev).
 
     [
       'boardSlottingMarshaller',
+      'fetchLatestEcQuestion',
```

Comment:
`readPublished` does this easily enough, not worth having a helper for every path

Reply:
Resolved in https://github.com/Agoric/agoric-sdk/pull/10385/commits/e1eaa5bf86ea936dee10bd996a60a49e5d157694